recipes/fsnviz/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+$PYTHON setup.py install 
+
+# Add more build steps here, if they are necessary.
+
+# See
+# http://docs.continuum.io/conda/build.html
+# for a list of environment variables that are set during the build process.
```

Comment:
Can you remove the comments from here? (https://bioconda.github.io/guidelines.html#comments-in-recipes)